python/lang/correctness/return-in-init.py

```diff
@@ -36,3 +36,9 @@ def __init__():
     # OK, not in class
     yield 5
 
+class Ab:
+    def func1():
+        if not hello:
+            yield from blah
```

Comment:
I'm surprised this parses... `from` is a keyword that usually only works in imports.